README.md

```diff
@@ -24,39 +24,75 @@ We are looking for posts that inform, teach, experiment, and generally enrich ou
+- If you are submitting an article or presentation that already exists, please send it in its entirety with a note on the expressed permission to publish from the owner of the content.
+- Once your submission has been approved, it will be added to the Node.js Foundation Medium publication publishing calendar and a publish date will be provided, so you may plan to promote accordingly through your personal and company’s social media channels.
 
 ## Review Process for Original Posts
-- The review team will provide comments and edits. 
-- Once the review team has made suggested edits, they will share all suggested edits with the author who has final sign off on content.
-- If the author feels comfortable, we would also like to list out the names of the technical editors of the post.
-- The review team will share all suggested edits with the author, who has final sign off on content.  
-- Zibby Keaton will then post to the Node.js Foundation Medium publication.
-- The review process may take between two to three weeks depending on the number of edits and revisions involved. 
-
-## Review Process for Created Medium Posts
-- Members from the evangelism working group are invited to become editors on Medium. If interested, contact Zibby Keaton here: zkeaton@linuxfoundation.org.
-- Editors are allowed to request to put already available public content on the Node.js Foundation Medium publication if they follow the above guidelines. 
-- If you are interested in becoming a technical editor, please file an issue.
+- Writers submit an abstract via email at medium@nodejs.org.
+
+- Editors have one week to provide approval or feedback - this is done internally on the alias and not to be shared with the writer until a final decision and suggested changes are reached as a group.
+
+- If the abstract is approved, the writer will be informed and a task will be created on the [Trello board](https://trello.com/b/M9VUQl9S/nodejs-collection) for the post
+
+- The contributor will be made a “Writer” on the Node.js Collection medium account.
+
+- The contributor will write the first draft and Write will submit this draft on Medium and an editor or two will be assigned to review. The contributor will ping the medium alias to alert the editors a draft has been submitted. Clear directions on how to submit will be provided to the contributor.
+
+- Editor from the Node.js Collection will review the post and provide the writer comments or feedback. This process can take up to a month to complete.
+
+- The writer to approve all content and send a note to medium@nodejs.org once finished.
+
+- Once the piece is complete, and the writer approves the content, an editor from Node.js PR team to review one final time and accept into publication.
+
+- Once the submission has been approved, it will be added to the Node.js Collection Medium publication - a publication date will be provided for proper social planning/ promo.
```

Comment:
Additional unnecessary space after the slash in this line (`planning/ promo.`)

Reply:
```suggestion
- Once the submission has been approved, it will be added to the Node.js Collection Medium publication - a publication date will be provided for proper social planning/promo.
```